dockerize.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# convenience script for creating/deleting docker files after compiling services
+
+make_images () {
+    services=("eureka-server" "config" "Spring-Cloud-Gateway" "accounts" "inventory" "commerce")
+
+    for ((i=0; i<${#services[@]}; i++)); do
+        cd ${services[i]}
+        #mvn clean package
+        if [ ${services[i]} = "Spring-Cloud-Gateway" ]; then
+            echo 1
+            docker build -t 'aypas/spring-cloud-gateway' .
+        else
+            echo 2
+            docker build -t aypas/${services[$i]} .
+        fi
+        echo `ls`
+        cd ..
```

Comment:
btw, we have commands `pushd` and `popd` which can fit in well in place of `cd` in cases like this. just a tip not a needed change.

Reply:
im gonna remove this file. didn't intend it to get in the commit.